### ISSUE TITLE
form-builder tree-view accessibility issues

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#6163]
Here is a list of issues from accessibility testing. 

After keying DEL on an item the delete dialog shows. After keying enter to delete the item the dialog closes but does not return focus to the tree view.

"Delete this page?" dialog
- clicking buttons "cancel" or "confirm"  does nothing in Edge (Windows)
- keying enter on buttons "cancel" or "confirm"  does nothing
- Closing the dialog by "cancel" does not return focus to the tree view item that was previously focussed
- Closing the dialog by "confirm" does not return focus to the tree-view or similar
- ESC does not close the dialog

Drag and Drop
- Item entering "Drag state" is not announced (VO only)
- Item exiting with a "Drop state" is not announced
- Item exiting with a "Drop state" is not announced

**Comment by thiessenp-cds** [comment]
Issue was resolved in a past fix.
